lua/obsidian/init.lua

```diff
@@ -197,7 +197,7 @@ client.new_note = function(self, title, id)
 
   -- Get path.
   ---@type Path
-  local path = Path:new(self.dir)
+  local path = fldr == nil and Path:new(self.dir) or Path:new(fldr)
```

Comment:
What happens if `opts.notes_subdir` is set, and `fldr` is equal to that directory? I think we'd want to check for that.

For example, my vault is at `~/epwalsh-notes`, and I have `opts.notes_subdir` set to `notes/`. So if I'm editing a file in `~/epwalsh-notes/notes/` and autocomplete a new note, I think `fldr` here would be `~/epwalsh-notes/notes/`, so below we'd set `path` to `~/epwalsh-notes/notes/notes/`.

Reply:
Yes, I should have thought of this one. Changed it so if the dir is given `notes_subdir` is ignored.